src/styles/HomePage.module.css

```diff
@@ -0,0 +1,231 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+/* Books display section */
+.ul {
+  border-bottom: 1px solid #e8e8e8;
+  padding: 0 0 2.5rem;
+  margin: 2.313rem 6.25rem 1.813rem;
+  display: flex;
```

Comment:
- [ ] You have this style repeated on 9, 18, 85, 92, and 173 whereas your CSS code should be a DRY code, would you please refactor your code in order to avoid repeating yourself? You can refer to this article on how to achieve that: [The Good & Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)